lib/imgix/rails/tag.rb

```diff
@@ -4,72 +4,26 @@ class Imgix::Rails::Tag
   include Imgix::Rails::UrlHelper
   include ActionView::Helpers
 
-  def initialize(path, source: nil, tag_options: {}, url_params: {}, widths: [])
+  def initialize(path, source: nil, tag_options: {}, url_params: {}, srcset_options: {})
     @path = path
     @source = source
     @tag_options = tag_options
     @url_params = url_params
-    @widths = widths.length > 0 ? widths : target_widths
+    @srcset_options = srcset_options
   end
 
 protected
 
-  def srcset(url_params: @url_params, widths: @widths)
-    widths = widths || target_widths
+  def srcset(source: @source, path: @path, url_params: @url_params, srcset_options: @srcset_options, tag_options: @tag_options)
+    params = url_params.clone
 
-    srcset_url_params = url_params.clone
-    srcsetvalue = widths.map do |width|
-      srcset_url_params[:w] = width
+    width_tolerance = ::Imgix::Rails.config.imgix[:srcset_width_tolerance]
```

Comment:
Pulling a config value in from another library feels a little odd to me.  Can we just leave this option unset and let Rails apply its default as normal?

Perhaps an even better question: is there a good reason this option isn't available in `imgix-rails` like the others are?

Reply:
I'm not totally sure what you mean here. The `srcset_width_tolerance` is defined at at the application config level (same as `source`, `sign_key`, etc). I admit this is a little strange just because it's a configuration that applies to a very specific subset of functionality, but I also do like that it allows users to set it once and forget about it.

Reply:
Ahh, my bad. Ignore me, I totally misread this. I agree that this should be an application config, not something the user has to specify for every image.